### PR TITLE
allow admins to specify # of hotel rooms for band contracts

### LINKED ISSUE #57: as a music dept head I want to specify the number of hotel rooms per band
Currently on the band agreement this is hardcoded at 1 room, but we might want to specify on a per-band basis either 0 rooms or more than 1 room.

image

### PR DESCRIPTION
**Pull request by binary1230**
fixes #57 

Previously, our contracts always assumed we were offering hotel rooms to each band.  Now, make this configurable for 0 or more hotel rooms.

What it looks like:

image

This also affects the band contract and the "hotel" section will be hidden if we are offering no rooms.

### CODE REVIEW

**Comment by binary1230** [comment]
DO NOT MERGE YET, THIS REQUIRES A DB MIGRATION 

hi @RobRuana  plz2halp

**Comment by binary1230** [comment]
db migration info:   this adds a new column for # of hotel rooms, the default value should be made to be '1' for any existing bands already in the system.

File: bands/templates/bands/agreement-base.html [hunk 1]
```diff
@@ -84,7 +86,9 @@
     </h3>
     <p>
         {% block band_text_parking_inner %}
-        To be included with hotel room for <b>{{ band.vehicles }} vehicle{{ band.vehicles|pluralize }}</b>.
+        To be included
```
Comment (kitsuta) [inline bands/templates/bands/agreement-base.html]: Business logic sanity check: this text could end up saying "To be included for 4 band vehicles." Does this mean we'll provide parking with or without a hotel room?

Comment (binary1230) [inline bands/templates/bands/agreement-base.html]: yea, this is my interpretation based on how we've done it in the past

if there's a different policy later, supermag folks will let us know

**Pull request merged**